### ISSUE TITLE
Textfield label does not disappear when text is entered

### ISSUE THREAD

**Issue by kittykatattack** [debois/elm-mdl#279]
Hello, 

I'm experience this strange effect with 8.1:

![screen shot 2017-01-30 at 3 53 23 pm](https://cloud.githubusercontent.com/assets/2936104/22441222/83b6e32c-e704-11e6-9157-9c358472f160.png)

Here's the code that produces this:
```
mdlTextfield id msg =
      Textfield.render MDL [ id ] model.mdl
      [ Textfield.autofocus
      , Textfield.maxlength 1000 
      , Textfield.label "Enter words here..."
      , Options.on "input" (Json.map msg targetValue)
      , Options.input
        [ -- css "font-size" "1.3em"
        ]
      ]
      []

```
Has anyone else experienced this?
It might just be something quirky with my installation... ?

**Comment by nilsga** [comment]
Sounds like it could be a similar issue to https://github.com/debois/elm-mdl/issues/278

**Comment by Matsebassen** [comment]
I'm having the same issue. It's very strange, I hope there will come a fix for this. 
And yes, it's similar to the issue with floating labels

**Comment by aforemny** [comment]
I will close here. Continuation (and fix) in #333. Thanks for all the input on this!
